zio-http/src/main/scala/zio/http/Scheme.scala

```diff
@@ -21,10 +21,11 @@ import zio.stacktracer.TracingImplicits.disableAutoTrace
 
 sealed trait Scheme { self =>
   def encode: String = self match {
-    case Scheme.HTTP  => "http"
-    case Scheme.HTTPS => "https"
-    case Scheme.WS    => "ws"
-    case Scheme.WSS   => "wss"
+    case Scheme.HTTP            => "http"
+    case Scheme.HTTPS           => "https"
+    case Scheme.WS              => "ws"
+    case Scheme.WSS             => "wss"
+    case Scheme.ChromeExtension => "chrome-extension"
```

Comment:
I am not sure, that having this explicitly is a good idea. Maybe we should have a `Custom` case, like for http method?
@vigoo @jdegoes

Reply:
Not sure if this is the right solution also, but, as it is today, zio http fails to accept requests from chrome extensions